papis/commands/config.py

```diff
@@ -1,34 +1,60 @@
 """
-The command config is a useful command because it allows you to check
-the configuration settings' values that your current `papis` session
-is using.
+The ``config`` command allows you to query the settings used by Papis on your
+system.
 
-For example let's say that you want to see which ``dir`` setting your
-current library is using (i.e., the directory or the dir that appears
-in the definition of the library in the configuration file), then you
-would simply do:
+The ``config`` command returns the value used by Papis. Therefore, if you
+haven't customized some setting, it will return the default value. In contrast,
+if you have customized it, it will return the value set in the configuration
+file. To show the list of default and overwritten configuration values in a
+section use
+
+.. code::
+
+    papis config --default --section <name>
+    papis config --section <name>
+
+or call ``papis config`` without the section argument to show the settings
+available for all the known sections.
+
+We can also query specific settings. Let's say you want to see which ``dir``
+setting your default library is using. You can achieve this with:
 
 .. code::
 
     papis config dir
 
-If you wanted to see which ``dir`` the library ``books`` has, for example
-then you would do
+If you wanted to see which ``dir`` the ``books`` library uses, you would do:
 
 .. code::
 
     papis -l books config dir
 
-This works as well for default settings, i.e., settings that you have not
-customized, for example the setting ``match-format``, you would check
-it with
+With ``-l``, Papis selects a specific library (here, the "books" library). The
+rest works just as above.
+
+Settings from a specific section in the configuration file can also be
+accessed. To take an example, the `Bibtex`_ command's settings can be
+accessed with
 
 .. code::
 
-    papis config match-format
-    > {doc[tags]}{doc.subfolder}{doc[title]}{doc[author]}{doc[year]}
+    papis config --default --section bibtex default-read-bibfile
+    papis config --section bibtex default-read-bibfile
 
-You can find a list of all available settings in the configuration section.
+For some more advanced usage, we can also query multiple settings at once. Then
+it may be useful to overwrite the selected section for several of the settings.
+This can be achieved by
```

Comment:
```suggestion
Alternatively, you can also you the ``<section.setting>`` format to query the value of
a setting in a specific `section`:

.. code::
    papis config bibtex.default-read-bibfile
    papis config --default bibtex.default-read-bibfile
    
For some more advanced usage, we can also query multiple settings at once.  Here,
sections specified with ``<section>.<setting>`` override the section specified by
``--setting <section>``. This can be achieved by:
```